README.md

```diff
@@ -1,70 +1,113 @@
-# Getting Started with Create React App
+# Spaces Traveler's Hub
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+![React](https://img.shields.io/badge/-React-61DAFB?logo=react&logoColor=white&style=for-the-badge)
+![Redux](https://img.shields.io/badge/redux-%23593d88.svg?style=for-the-badge&logo=redux&logoColor=white)
+![React Router](https://img.shields.io/badge/React_Router-CA4245?style=for-the-badge&logo=react-router&logoColor=white)
+![Bootstrap](https://img.shields.io/badge/bootstrap-%23563D7C.svg?style=for-the-badge&logo=bootstrap&logoColor=white)
 
-## Available Scripts
+> Web application that allows users to book rockets, dragons, and join selected space missions using data from the SpaceX API.
 
-In the project directory, you can run:
+![screenshot](Animation02.gif)
 
-### `npm start`
+## Live Demo
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+[![Netlify Status / heroku](xxxxxxxxxxxxxx/)
```

Comment:
- [ ] The `live link` and the `MIT link` in the README file are broken. I would highly recommend fixing those sections to make README professional.

Reply:
MIT link is fixed

Reply:
All good now 👍🏽